gravitee-apim-console-webui/src/services-ngx/api-v2.service.spec.ts

```diff
@@ -578,7 +578,7 @@ describe('ApiV2Service', () => {
         })
         .flush(fakeApi);
 
-      expect(done).toEqual(['last api fetch', 'last api fetch', 'api refreshed']);
+      expect(done).toEqual(['last api fetch', 'api refreshed']);
```

Comment:
why do we lose one entry ?

Reply:
the question is more why there were 2 before 🙈

Reply:
i will ad some more test

Reply:
AAA and I knew it from yesterday 
It's normal because the 1st getLastApiFetch is still under subscription. 
And in done we have the last 2 api fetches and the refresh info.

and now that the 2 api are equal, well, just one event. 

i'm going to add some more test